client/src/components/Button/index.tsx

```diff
@@ -2,14 +2,15 @@ import React, { FC, ReactNode } from 'react';
 import styles from './styles.module.scss';
 
 enum ButtonStyles {
-  themeBtn = 'themeBtn',
-  iconBtn = 'searchBtn'
+  ThemeBtn = 'themeBtn',
+  IconBtn = 'searchBtn',
+  FormBtn = 'formBtn'
 }
 interface ButtonProps {
   type: 'button' | 'reset' | 'submit' | undefined;
   children: ReactNode;
   className: string;
-  onClick?: () => void;
+  onClick?: (event: any) => void;
```

Comment:
I think it would be better to define type, i.e. `(event: MouseEvent<HTMLButtonElement>)` .

Reply:
Actually I have to write  (event: FormEvent<HTMLButtonElement>)